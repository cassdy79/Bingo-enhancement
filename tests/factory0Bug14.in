1
a
b
1 R 1
2 R 1
3 U 2
4 U 2
5 Y 3
save factory0Output
4