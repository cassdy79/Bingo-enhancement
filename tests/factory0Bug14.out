a
b
0
0
0
FBBBUUYBYYRRYBLL
....
....
....
....
....
byrul
lbyru
ulbyr
rulby
yrulb
byrul
lbyru
ulbyr
rulby
yrulb
R
U.
Y..
....
.....
R
U.
...
....
.....
       
       

RUBBBRLBBRYYLBLBYRYUULYRUYYRUULLLLYBUBURBLRBRRUYURLLYURYUBLULRYBUULYYRRYULBBRLLL
14